celest/coordinates/frames/attitude.py

```diff
@@ -0,0 +1,109 @@
+
+
+from celest.coordinates.frames.base_positions import Position3d
+from celest.coordinates.ground_location import GroundLocation
+from celest.file_save import TextFileWriter
+from celest.units.core import Unit
+from celest.units.quantity import Quantity
+from celest import units as u
+import numpy as np
+
+
+class Attitude(Position3d):
+    """Satellite attitude.
+
+    The attitude of a satellite is defined by the roll, pitch, and yaw angles
+    that take the satellite from the lvlh frame to a ground target pointing
+    orientation.
+
+    Parameters
+    ----------
+    julian : np.ndarray
+        1-D array containing time in the J2000 epoch.
+    roll, pitch, yaw : np.ndarray
+        1-D array containing the coordinate data.
+    unit : Unit
+        The unit of the spatial data.
+    location : GroundLocation
+        The ground location for the satellite to point to.
+
+    Attributes
+    ----------
+    roll, pitch, yaw : Quantity
+        Coordinate data.
+    location : GroundLocation
+        The ground location for the satellite to point to.
+    time : Quantity
+        Times associated with coordinate dimensions.
+
+    Methods
+    -------
+    save_text_file(file_name)
+        Save data as a pretty text file.
+
+    See Also
+    --------
+    AzEl : Azimuth elevation coordinates.
+    GCRS : Geocentric Celestial Reference System.
+    ITRS : International Terrestrial Reference System.
+    LVLH : Local vertical local horizontal coordinates.
+    WGS84 : Geographical coordinates.
+
+    Examples
+    --------
+    Let `julian`, `roll`, `pitch`, and `yaw` be `np.ndarray` instances and
+    `location` be a `GroundLocation` instance. An `Attitude` object can be
+    initialized:
+
+    >>> attitude = Attitude(julian, roll, picth, yaw, u.deg, location)
+
+    Save `Attitude` data to a text file:
+
+    >>> attitude.save_text_file("attitude_data")
+    """
+
+    def __init__(self, julian: np.ndarray, roll: np.ndarray, pitch: np.ndarray,
+                 yaw: np.ndarray, unit: Unit, location: GroundLocation) -> None:
+
+        super().__init__(roll, unit, pitch, unit, yaw, unit, julian, u.jd2000)
```

Comment:
These are changes for the `Position3d` object: change the constructor  of the super class to have consistency with the `Attitude` constructor.

```suggestion
        super().__init__(julian, u.jd2000, roll, unit, pitch, unit, yaw, unit)
```

Ditto for the `Position2d` class.

Reply:
Ahh this really isn't necessary.